README.md

```diff
@@ -337,6 +337,64 @@ are done with the migration.
 
   Modify this column if desired.
 
+### `partitions` ###
+
+- `schema` (type `name`): schema of the table with the partition
+
+- `table_name` (type `name`): name of the related table
+
+- `partition_name` (type `name`): name of the partition
+
+- `orig_name` (type `name`): name of the partition in the remote source
+
+- `type` (type `text`): one of the supported partitioning methods `LIST`,
+  `RANGE` or `HASH`
+
+- `expression` (type `text`): partitioning expression used to define a table's
+  partitioning
+
+- `position` (type `integer`): defines the order of the table partitions (must
+  start at 1)
+
+- `values` (type `text`): for a `RANGE` method, contains the upper bound value,
+  for a `LIST` method, contains a list of comma-separated values
+
+- `is_default` (type `boolean`, default `FALSE`); `TRUE` if it is the default
+  partition 
+
+- `migrate` (type `boolean`, default `TRUE`): `TRUE` if the constraint should
+  be migrated
```

Comment:
This looks like it was copied from the constraints; should be "partition".

Reply:
Good catch, especially for `RANGE` partitioning. 

Another way to transform a partitioned table as a simple table could be simply remove all partition specifications from `plsql_stage.partitions` related to source table. But, problem still there if only one partition is deleted from scheme.

Reply:
I just realized that there is another problem.

When the data are migrated, you just select from the partitioned table, that is, you get all data from all partitions, right? That would fail to load on PostgreSQL if you omitted one partition.

I am not sure what to do about that. I have two ideas:

1. Change the way data are migrated so that it is partition by partition (that is the hardest solution).
2. Only provide the option to migrate partitioning with all partitions as they are or not migrate partitioning at all. That would be good enough from my point of view.
   Then we should do away with the `migrate` column for partitions. We can either introduce a `migrate_partitioning` flag on the table or expect that the user deletes the partitions from the migrator table.